### PR TITLE
feat - added alert component

### PR DESCRIPTION
**Pull request by shubhambishnoi**
Screenshot from 2022-02-05 11-58-08
Screenshot from 2022-02-05 12-03-09

### CODE REVIEW

**Comment by xsaurabhsuthar** [comment]
Can you please give the updated route for previews?

**Comment by xsaurabhsuthar** [comment]
Alert aren't looking very good. You can check readability using https://coolors.co/contrast-checker/112a46-acc8e5.

**Comment by shubhambishnoi** [comment]
> Alert aren't looking very good. You can check readability using https://coolors.co/contrast-checker/112a46-acc8e5.

pls check now ...made text bold.. and gave lighter shade of colors

**Review by xsaurabhsuthar — APPROVED** [review]
Looks good.

File: templates/installation.html [hunk 1]
```diff
@@ -48,7 +48,7 @@ <h3>Installation Guide</h3>
             <p>In order to start using Escape UI components in your projects you can either import the below</p>
             <p>given url in your css file or link it to your html file.</p>
             <iframe
-                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253C--------%2520put%2520this%2520in%2520your%2520html%2520-----------------%253E%250A%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fescapeui.netlify.app%252Fstyling%252Fcomponents.css%2522%253E%250A%250A%253C------------%2520put%2520this%2520in%2520your%2520css%2520---------------%253E%250A%2540import%2520url%28%2522https%253A%252F%252Fescapeui.netlify.app%252Fstyling%252Fcomponents.css%2522%29%253B"
+                src="https://carbon.now.sh/embed/vP9R42dyQZO211ZMCvQj"
```
Comment (saketkum11) [inline templates/installation.html:51]: can you remove the space:

**Review by saketkum11 — COMMENTED** [review]
Good to go ahead.

**Review by saketkum11 — APPROVED** [review]

**Pull request merged**